test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+odoo-addon-rental_base @ git+https://github.com/OCA/vertical-rental.git@refs/pull/48/head#subdirectory=setup/rental_base
```

Comment:
Please put this in a separate commit. See: https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29